src/irep2/irep2_expr.cpp

```diff
@@ -350,6 +350,68 @@ expr2tc constant_string2t::to_array() const
   return final_val;
 }
 
+#ifndef NDEBUG
```

Comment:
Do we really need this? If so, can you add some comments for this?

Reply:
This one we don't need, I've removed it.